### ISSUE TITLE
Accessible by multiple analysts

### ISSUE THREAD

**Issue by geekymitch** [yeti-platform/yeti#294]
Before opening an issue, please make do a quick search through the current issues
and the mailing list (https://groups.google.com/forum/#!forum/yeti-users) to see
if it hasn't been answered already.

### Description
Looking to make existing install of yeti accessible for multiple users (so they all work off the same instance in other words)

### Environment

OSX

#### Expected behavior
Just not sure how to go about setting this up - I'm assuming I'll need to get myself a static IP in my work environment, which is no issue...but ideally I'd like to be able to access the same instance of yeti, but not necessarily use a remote connection...

I'm new at this, so step by step would be greatly appreciated if possible - thank you!

**Comment by tomchop** [comment]
Unfortunately this depends a lot on your environment, so providing step-by-step instructions will be difficult. 

Yeti listens on all interfaces by [default](https://github.com/yeti-platform/yeti/blob/master/yeti.py#L23), so you should be all set by pointing your web browser to the IP of the computer Yeti's running on and the corresponding port number (default is 5000)

**Comment by geekymitch** [comment]
I can probably manage that - so I don't actually need to go through the
yeti install on the separate machine, just need a static IP etc.

If I DO need to change ports (a possibility) how would I go about that?

Thanks again - and if you have a second please do send me over a slack
invite - appreciate it greatly!
- Mitch

**Comment by tomchop** [comment]
Yeah, Yeti is a web app so one install makes it available for everyone that can reach that server. :)

I think adding a port parameter should do it, e.g. `webapp.run(host="0.0.0.0", port=8080)` to run it on port 8080

**Comment by geekymitch** [comment]
Sorry to bother you again, but could you expand on this just a little bit?
Have it running on my box, have my IP address, now how do I put the two
together?    i.e. not sure where I designate the ip/port combination before
running things.

Sorry again for all the questions and thank you for your patience.
- Mitch

**Comment by gaelmuller** [comment]
This was answered, please reopen if needed.
